### ISSUE TITLE
Page freezes when trying to insert Gutenberg block WP 6.2

### ISSUE THREAD

**Issue by zidancek** [WordPress/gutenberg#50468]
### Description

I recently updated my website to WP 6.2. Ever since then the page freezes whenever I try to insert new Gutenberg block (I have tried multiple different blocks, paragraph, gallery, more...) Every time the page becomes unresponsive and crashes. I have tried in Chrome and in Brave browsers, both the same. 

What could be the cause and is there a quick fix for this? 

Thank you!

### Step-by-step reproduction instructions

Click on the plus to add a new Gutenberg bloc. The page freezes then.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by zidancek** [comment]
Update. It works well on Safari browser...

**Comment by ramonjd** [comment]
Could you confirm which version of the Gutenberg plugin you're running? 

Are there any themes or other plugins installed and activated as well? Any errors in the console?

**Comment by kevsnewsmail** [comment]
I am having exactly the same problem but it only happens when I am working in Google Chrome. I have tried it in MS Edge (I am using windows) and it work fine, I have also tried it in chrome on a different PC and it works fine there to, except that PC has an older version of Chrome installed, so it points to this being an issue with Chrome. 

Also, it only happens when I rollover the icons in the Gutenberg window prompting the info box to appear.

I will use MS Edge for now, perhaps this is the thin edge of the wedge to make m an Edge user?
